src/backend/base/langflow/components/nvidia/__init__.py

```diff
@@ -1,4 +1,11 @@
 from .nvidia_ingest import NvidiaIngestComponent
 from .nvidia_rerank import NvidiaRerankComponent
 
-__all__ = ["NvidiaIngestComponent", "NvidiaRerankComponent"]
+try:
+    import gassist.rise  # noqa: F401
+
+    from .system_assist import NvidiaSystemAssistComponent
+
+    __all__ = ["NvidiaIngestComponent", "NvidiaRerankComponent", "NvidiaSystemAssistComponent"]
+except ImportError:  # skip the component if gassist is not installed (sys_platform != win32)
+    __all__ = ["NvidiaIngestComponent", "NvidiaRerankComponent"]
```

Comment:
If we know the condition shouldn't we just check for win32?

Reply:
okay, sure